db/migrate/20200212122000_add_service_parameters_raw.rb

```diff
@@ -0,0 +1,5 @@
+class AddServiceParametersRaw < ActiveRecord::Migration[5.1]
+  def change
+    add_column :order_items, :service_parameters_raw, :jsonb
```

Comment:
Is there any value pre populating this with existing raw parameters and simulaneously populating `service_parameters` with the masked values?

Reply:
Since we do not have anything supported in prod yet I'd prefer the migration just duplicate `service_parameters` to `service_parameters_raw` in case something eventually expects the new column to be populated.  Dealing with tenants and having to pull the survey for each element seems like a lot of overhead that does not buy us much.

We can add a single line to the migration to duplicate the parameters:
`OrderItem.update_all("service_parameters_raw=service_parameters")`

Thoughts?